.swiftlint.yml

```diff
@@ -6,11 +6,14 @@ excluded:
   - DerivedData
 
 disabled_rules:
+  - function_parameter_count
+  - nesting
```

Comment:
It is triggered on typealises and other types nested more than 1 levels deep. It's not something we will be able to fix soon and without breaking changes.
![Screenshot 2023-02-03 at 17 00 00](https://user-images.githubusercontent.com/36865532/216635756-04d2bcee-7bb8-4044-87a9-49691d91f5b5.png)

Reply:
Done.